Makefile

```diff
@@ -13,6 +15,6 @@ test:
 	./test.sh
 
 install:
-	createdb userdb 
-	psql -v ON_ERROR_STOP=1 -h localhost -d userdb -f schema.sql
+	createdb userdb
+	@(foreach file,$(SCHEMA_FILES),psql -v ON_ERROR_STOP=1 -h localhost -d userdb -f $(file);)
```

Comment:
Instead of running multiple times, could you just do: `cat $(SCHEMA_FILES) | psql`
I'm wondering if we can do this all in a single transaction.